.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+    "recommendations": [
+        "ms-vscode.cpptools",
+        "ms-vscode.cmake-tools",
+        "marus25.cortex-debug",
+        "twxs.cmake"
```

Comment:
Without the other PR this probably doesn't work from this branch right?
Is there a way to indicate "not recommended" extensions?

Reply:
Yeah, we would be technically recommending extensions that would show broken functionality without the other PR being merged. 

For non-recommended extensions you'd use `unwantedRecommendations`.

Reply:
Okay, so for this PR let's add the CMake extension to the unwanted, and then add it in the other PR